<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SSG</class>
 <widget class="QMainWindow" name="SSG">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SSG</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="menu">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>메뉴</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="menuBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>111</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="map">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>지도</string>
     </property>
    </widget>
    <widget class="QPushButton" name="mission">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>미션</string>
     </property>
    </widget>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>저장</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>종료</string>
     </property>
    </widget>
    <widget class="QPushButton" name="time">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>시간표</string>
     </property>
    </widget>
    <widget class="QPushButton" name="out_btn">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>나가기</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="chat">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>691</width>
      <height>171</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="missionBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>401</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>미션!</string>
    </property>
    <widget class="QLabel" name="chat_m">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>331</width>
       <height>61</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::Box</enum>
     </property>
     <property name="lineWidth">
      <number>3</number>
     </property>
     <property name="midLineWidth">
      <number>3</number>
     </property>
     <property name="text">
      <string>도서관에 가서 프린터기를 찾으세요</string>
     </property>
    </widget>
    <widget class="QPushButton" name="OK">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>확인</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Yes">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>120</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>예</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="mission_show">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="lineWidth">
     <number>3</number>
    </property>
    <property name="midLineWidth">
     <number>3</number>
    </property>
    <property name="text">
     <string>도서관에 가서 프린터기를 찾으세요</string>
    </property>
   </widget>
   <widget class="QPushButton" name="next">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>다음</string>
    </property>
   </widget>
   <widget class="QPushButton" name="go">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>510</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>확인</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>651</width>
      <height>311</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="front">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>52</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="left">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="right">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>75</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mapclose">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>340</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>지도 닫기</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
